console/account/how-to/report-incident.mdx

```diff
@@ -0,0 +1,63 @@
+      The different types of content or illegal behavior that can be reported is described in Scaleway’s [Abuse notice](https://www.scaleway.com/en/abuse-notice/).
+    </Message>
+    <Message type="important">
+      We hereby remind you that reporting content or activity as illegal in order to obtain its removal or to stop its distribution, when you do know the information are indeed inaccurate, is punishable by one year imprisonment and a €15,000 fine.
+    </Message>
+6. Enter your email address and name if required to do so.
+7. Describe what you have observed in as much detail as possible. Provide information such as logs, time zones, headers, the precise location and if applicable, the email addresses to which the content is made accessible.
+    <Message type="important">
+      This description will be directly forwarded to our customer. Do not therefore include any confidential or personal information which you do not want them to have access to. 
+      Scaleway denies all responsibility for the use which could be made by third parties of any information that you provided in this field.
+    </Message>
+8. Check the box confirming that you are submitting this report in good faith.
+9. Once you have entered the information mentioned above, check the CAPTCHA form and click **Submit abuse report**.
+    <Message type="note">
+      In case you encounter difficulties to submit an abuse, you can contact [Scaleway’s abuse team](https://www.scaleway.com/fr/contact-trust-et-safety/).
+    </Message>
+
+## How is my report processed?
+Once you have completed the [report abuse form](https://abuse.scaleway.com/), you will receive an email confirming its reception by Scaleway’s abuse team.
+
+Scaleway’s abuse team will verify the exactness of the report, that all the necessary information has been provided for the report to be processed. They will also verify whether the report indeed relates to explicitly illegal behavior or content with regard to French regulations and the evidence and information you have provided. If your report meets the above criteria, it will be validated by the abuse team and sent to Scaleway’s customer so that the necessary corrective measures can be taken.
+
+1. You will receive updates regarding the resolution of the abuse via the email address provided in the form if you provided one.
+    <Message type="note">
+      Should Scaleway’s customer not respond, or should the transferred report not be processed correctly, appropriate measures will be taken by Scaleway. These measures may vary from a simple warning to termination of all services of said customer.
+    </Message>
+2. If your report is rejected, you will be informed of the reasons why by email.
+    <Message type="note">
+      * Massively reporting the same practices or content forces the abuse team to carry out unnecessary actions which have a negative impact on their work. As such, it is counterproductive to massively report the same practices or content.
```

Comment:
Not sure about "massively" - maybe "repeatedly"?

Reply:
```suggestion
      * Repeatedly reporting the same practices or content forces the abuse team to carry out unnecessary actions which have a negative impact on their work. As such, it is counterproductive to massively report the same practices or content.
```